app/views/vacancies/_filters.html.haml

```diff
@@ -1,7 +1,7 @@
 - content_for :advanced_filters do
   .govuk-form-group
     = label_tag 'radius', t('jobs.filters.radius'), class: 'govuk-label'
-    = select_tag 'radius', options_for_select(radius_filter_options, radius), class: 'govuk-select form-control form-control-block'
+    = select_tag 'radius', options_for_select(radius_filter_options, radius), class: 'govuk-select form-control form-control-block govuk-!-width-full'
```

Comment:
I took a look at the classes here, because I wasn't familiar.  The `form-control-block` seems to be undefined?  The `form-control` makes the field grey?  Not sure we want either.  Not something that needs to be addressed in this PR, but might be worth fixing this in a follow up one.

Reply:
As a general heads up for this upgrade, did you check the mobile version looks good too?

Reply:
Yep ran through the pages on mobile and nothing was borked. The hiring staff manage jobs view isn't great on mobile but that was also the case before the upgrade - mainly because hiring staff wouldn't really use mobile to do the job listings

Reply:
will remove the `form-control-block` class too 👍

Reply:
@adrianclay code is on Edge if you wish to double check the mobile view 😄

Reply:
removed `form-control-block` in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1057/commits/c28f9e43959456e9bea5a5b3fd9c2fb9a90c3c0d